test/ApcTokenStorageTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace {  
```

Comment:
I believe this global namespace is redundant, PHP does not support "namespaced" variables.